test/AutoDiff/control_flow_sil.swift

```diff
@@ -172,7 +172,7 @@ func cond_tuple_var(_ x: Float) -> Float {
 // CHECK-SIL:   [[BB1_PRED:%.*]] = destructure_struct [[BB1_PB_STRUCT]]
 // CHECK-SIL:   copy_addr {{%.*}} to {{%.*}} : $*(Float, Float)
 // CHECK-SIL-NOT:   copy_addr {{%.*}} to {{%.*}} : $*Float
-// CHECK-SIL:   switch_enum [[BB1_PRED]] : $_AD__cond_tuple_var_bb1__Pred__src_0_wrt_0, case #_AD__cond_tuple_var_bb1__Pred__src_0_wrt_0.bb0!enumelt.1: bb5 // id: %81
```

Comment:
This comment should not have been checked in.  That said, this test failed because this instruction ID became `%77` instead of `%81`, which indicates that the new differentiation conditions eliminated some unnecessarily differentiated code.